tutorials/dataframe/df023_aggregate.C

```diff
@@ -0,0 +1,30 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// \notebook
+/// This tutorial shows how to use the Aggregate action to fill a TGraph with a parabola
+/// \macro_code
+///
+/// \date July 2018
+/// \author Enrico Guiraud, Danilo Piparo CERN, Massimo Tumolo Politecnico di Torino
+
+void df023_aggregate()
+{
+   ROOT::EnableImplicitMT(2);
+   auto d = ROOT::RDataFrame(20).DefineSlotEntry("x", [](unsigned int, ULong64_t e) { return static_cast<double>(e); });
```

Comment:
please separate in two different lines, this works but it's a bit of a hack -- in principle the `RDataFrame` object would go out of scope and unleash hell (it doesn't because `DefineSlotEntry` is optimized to return the `RDataFrame` itself, but that's an implementation detail)

Reply:
Done!